lib/twilio-ruby/twiml/twiml.rb

```diff
@@ -37,6 +35,7 @@ def to_s(xml_declaration = true)
         return ('<?xml version="1.0" encoding="UTF-8"?>' + xml) if xml_declaration
         xml
       end
+      alias to_xml to_s
```

Comment:
As this is Ruby, it is interpreted and run in order. At the point in the class definition where `to_xml` is aliased to `to_s`, `to_s` had been defined by an ancestor class and `to_xml` took on that definition. To move it here means that `to_xml` takes on the correct defintion of `to_s`.

This didn't cause an issue in `client_capability.rb` because the `ClientCapability` class itself doesn't define `to_s`, it is taken from `BaseJWT` which was defined before `ClientCapability` and therefore `to_xml` takes on the right definition.

There are a couple of articles that describe how `alias` works, and compare to `alias_method`:
* http://blog.bigbinary.com/2012/01/08/alias-vs-alias-method.html
* https://ernie.io/2014/10/23/in-defense-of-alias/

They both unintentionally show one thing, when using `alias` Ruby developers do so straight after defining the method they are aliasing to something else. So this placement of `alias to_xml to_s` is both correct and more rubyish than before.

One other thing, the existence of `alias to_s to_jwt` in `ClientCapability` and `def to_s ; to_jwt ; end` in `AccessToken` shows that some more refactoring can be done. PR incoming.